### ISSUE TITLE
False positive: XYZ is unknown import symbol (reportGeneralTypeIssues)

### ISSUE THREAD

**Issue by cpitclaudel** [microsoft/pyright#2175]
**Describe the bug**

Pyright produces false positives when modules and constants are mixed with modules first in an `from . import` line.

**To Reproduce**

```console
$ pyright --version
pyright 1.1.63
$ mkdir /tmp/pyright; cd /tmp/pyright
$ mkdir src
$ echo "A = 1" > src/__init__.py
$ echo "B = 2" > src/b.py
$ echo "from . import b, A" > src/c.py
$ pyright src
stubPath /tmp/pyright/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 3 source files
/tmp/pyright/src/c.py
  1:18 - error: "A" is unknown import symbol (reportGeneralTypeIssues)
1 error, 0 warnings 
Completed in 0.522sec
```

**Additional context**

The problem goes away is constants are imported before modules:

```console
$ echo "from . import A, b" > src/c.py
$ pyright src
stubPath /tmp/pyright/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 3 source files
0 errors, 0 warnings 
Completed in 0.521sec
```

**Comment by erictraut** [comment]
You appear to be using pyright 1.1.63, which is now 98 versions out of date! Please update to a recent version. We fixed this issue eons ago. :)
